.github/workflows/main.yml

```diff
@@ -14,10 +14,22 @@ jobs:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
+    - name: Install deps
+      run: |
+        sudo apt install -y curl
+        echo "deb [arch=amd64] http://robotpkg.openrobots.org/packages/debian/pub $(lsb_release -sc) robotpkg" \
+          | sudo tee /etc/apt/sources.list.d/robotpkg.list
+        curl http://robotpkg.openrobots.org/packages/debian/robotpkg.key \
+          | sudo apt-key add -
+        sudo apt update
+        sudo apt install -y clang-tidy libeigen3-dev robotpkg-pinocchio robotpkg-qpoases
+
     - name: Lint
       run: |
-        sudo apt install -y clang-tidy
         cd $GITHUB_WORKSPACE/cpp
+        export LD_LIBRARY_PATH=/opt/openrobots/lib:${LD_LIBRARY_PATH}
+        export CMAKE_PREFIX_PATH=/opt/openrobots
+        mkdir build && cd build && cmake -DBUILD_WITH_PINOCCHIO=ON -DBUILD_WITH_qpOASES=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ..
```

Comment:
Is there any reason to use two CI ? I think one of them could do everything.

Reply:
I was just trying out the new Github Action to see if there is anything interesting. Nothing interesting so far. The Github server seems to be slower quite a bit as well. At this rate we might as well use CircleCI for linting for the C++ code.

Reply:
I think one CI is better on many aspects (less maintenance, less energy...).

Reply:
Sure, totally agree. I have the bad habit of having to try new things out... This time it quite clear that CircleCI is fully capable of doing whatever we need it to do.

Luckily it looks like they (CIs) are basically copying each other, so moving from on to another is not very difficult.